packages/edit-site/src/components/global-styles/screen-block.js

```diff
@@ -152,15 +216,15 @@ function ScreenBlock( { name, variation } ) {
 				<StylesBorderPanel
 					inheritedValue={ inheritedStyle }
 					value={ style }
-					onChange={ setStyle }
+					onChange={ onChangeBorders }
 					settings={ settings }
 				/>
 			) }
 			{ hasEffectsPanel && (
 				<StylesEffectsPanel
 					inheritedValue={ inheritedStyleWithLayout }
 					value={ styleWithLayout }
-					onChange={ onChangeDimensions }
+					onChange={ setStyle }
```

Comment:
This is an unrelated change but it's a small optim/typo fix (same for the next change)